src/pointerwrappers.jl

```diff
@@ -1,6 +1,11 @@
 module PointerWrappers
 
-# Pointer wrapper for conveniently accessing nested structures
+"""
+    PointerWrapper
```

Comment:
```suggestion
    PointerWrapper(p::Ptr)
```
We often document the constructor in the docstring of a type.

Reply:
Done in [ef3e0b8](https://github.com/trixi-framework/P4est.jl/pull/67/commits/ef3e0b81e229227d843ff5c702438ab72434fd65).